esphome/components/online_image/online_image.cpp

```diff
@@ -71,24 +71,16 @@ bool OnlineImage::resize_(int width_in, int height_in) {
     return false;
   }
   auto new_size = this->get_buffer_size_(width, height);
-  ESP_LOGD(TAG, "Allocating new buffer of %d Bytes...", new_size);
+  ESP_LOGV(TAG, "Allocating new buffer of %d Bytes...", new_size);
```

Comment:
I'd like to keep at least this one as "debug"; memory is still an issue for some users, and having a log explicitly mentioning how much memory is being allocated helps with debugging user issues.
```suggestion
  ESP_LOGD(TAG, "Allocating new buffer of %d Bytes...", new_size);
```